test/functional/lib/redirect-handler/blockOrObserve.test.ts

```diff
@@ -68,6 +100,7 @@ describe('lib/redirect-handler/blockOrObserve', () => {
       sinonSandbox.restore()
       browserMock.tabs.query.resetHistory()
       browserMock.tabs.reload.resetHistory()
+      browserMock.tabs.update.resetHistory()
```

Comment:
if we're using a spy/mock/stub that sinonSandbox is aware of, it should be able to clear them all out for us.